custom_dc/deploy_cloud.md

```diff
@@ -52,68 +52,65 @@ This procedure creates a "dev" Docker package that you upload to the Google Clou
-
-For future deployments, select **Edit & Deploy Revision** from this page to select a new repository\ container and locate the link to the running instance.
+This procedure shows you how to create a service and set environment variables using the Cloud Console. 
 
 See also [Deploying to Cloud Run](https://cloud.google.com/run/docs/deploying) for more information on all the options you may set on your service.
 
-## Load custom data to a deployed instance
-
-Once you have deployed a custom instance to Google Cloud, you can continue to update your custom data in two ways:
-
-- Load the data from a local running instance, as described in [Load custom data in Cloud SQL](/custom_dc/data_cloud.html#load-data-cloudsql)
-- Use the `/admin` page from the running Cloud app.
+1. Go to https://console.cloud.google.com/run/ for your project.
+1. Click **Create service**.
+1. Keep the default **Deploy one revision from an existing container image** enabled.
+1. In the **Container image URL** field, click **Select** to open the **Select container image** window.
+1. In the list of images that appears, navigate to container image you pushed in the previous step, highlight it and click **Select**.
+1. In the **Service name** field, enter a name for your service.
+1. In the **Region** field, select your [location](/custom_dc/data_cloud.html#location).
+1. Under **Authentication**, select the relevant option depending on whether your site will be public or not. If it is public, enable **Allow unauthenticated invocations**.
+1. Set the following options:
+   - **CPU allocation and pricing**: **CPU is always allocated**
+   - **Service autoscaling** > **Minimum number of instances**: **1**
+1. Expand **Container, Volumes, Connections, Security** and expand **Settings**, and set the following options:
+  -  **Resources** > **Memory**: **8 GiB**
+  -  **Resources** > **CPU**: **2**
+  -  **Task timeout** > **Number of retries per failed task**: **3**
+  -  **Revision autoscaling** > **Minimum number of instances**: **1**
+  -  **Revision autoscaling** > **Maximum number of instances**: **1**
+1. Disable **Startup CPU boost**.
+1. Click the **Variables and Secrets** tab.
+1. Click **Add variable**.
+1. Add the same environment variables, with the same names and values as you did when you created the [data management run job](/custom_dc/data_cloud.html#env-vars) You can omit the `INPUT_DIR` variable.
+1. Add a variable for the `MAPS_API_KEY` and set it to your Maps API key.
+1. Click **Done**.
+1. Click **Deploy**.
+
+Follow the screen output to see the status details of the operation. Once it completes, a link to the deployed image is listed at the top of the page. Click on the link  to see the running instance.
+
+## Manage the service
+
+Every time you make changes to the code and release a new Docker artifact, or restart the [data management job](/custom_dc/data_cloud.html#run-job), you need to restart the service as well. To do so:
```

Comment:
An overall note: all references to re*starting* the data container should be replaced with re*running* it.

Reply:
OK, will do.